sessionctx/variable/sysvar.go

```diff
@@ -1335,6 +1335,15 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeSession, Name: TiDBReadConsistency, Value: string(ReadConsistencyStrict), Type: TypeStr, Hidden: false,
```

Comment:
Need to hide this system variable from `show` statement.

Reply:
I remembered it's hidden before...

Reply:
I turned it on in manually testing :(